src/main/java/org/folio/rest/core/RestClient.java

```diff
@@ -71,7 +70,9 @@ public Future<JsonObject> handleGetRequest(String endpoint) {
    */
   public Future<Void> handlePutRequest(String endpoint, JsonObject recordData) {
     if (logger.isDebugEnabled()) {
-      logger.debug("Sending 'PUT {}' with body: {}", endpoint, recordData.encodePrettily());
+      logger.debug("Trying to update {} object with data: {}", endpoint, recordData.encodePrettily());
+//    }
```

Comment:
which type of pattern is best 
old logging pattern 
-  **sending HTTP.METHOD with body { }**

new logging pattern based on documentation 
- **Trying to verb [method] {} object with data: {}**
![image](https://user-images.githubusercontent.com/113523904/201312488-b6b71801-28a9-4910-983c-85f396f918fb.png)

Reply:
Should I leave sending pattern or should I change to trying to verb..... pattern

Reply:
This pattern looks better as for me:
logger.debug("Trying to update {} object with data: {}", endpoint, recordData.encodePrettily());